### ISSUE TITLE
Server disconnecting: The connection to the terminal's pty host process is unresponsive, the terminals may stop working

### ISSUE THREAD

**Issue by raphaelchinchilla** [julia-vscode/julia-vscode#3003]
Today I updated VS Code to version 1.70.1 on a windows machine. Since then, I regularly keep receiving this message:

> The connection to the terminal's pty host process is unresponsive, the terminals may stop working.

And about half the times the REPL gets back online, and half it stops working.

I am not sure whether other people are experiencing it, but I am filing this issue just in case. Please close it if you believe it is not related to the julia-vscode extension.

**Comment by pfitzseb** [comment]
It's unclear whether this is a VS Code or extension issue, but [disabling shell integration](https://discourse.julialang.org/t/vscode-repl-hanging-after-a-while/85585/11?u=pfitzseb) should fix it for now.

The next extension release will (once again) not support shell integration on Windows via https://github.com/julia-vscode/julia-vscode/pull/3000.

**Comment by tclements** [comment]
Can confirm that I am experiencing this issue with the REPL on VS code 1.70.1 and Window 10 Enterprise.

**Comment by pfitzseb** [comment]
Does disabling shell integration, as suggested above, fix the issue for you?

**Comment by tclements** [comment]
Seems to have fixed for now - will report back if I get another pty host error.

**Comment by raphaelchinchilla** [comment]
FYI: I downgraded back to 1.69 and it is working fine again

**Comment by braamvandyk** [comment]
Same issue, but I also have this error message in the terminal:

```
ERROR: LoadError: IOError: connect: no such file or directory (ENOENT)
Stacktrace:
 [1] wait_connected(x::Base.PipeEndpoint)
   @ Sockets C:\Users\Braam\.julia\juliaup\julia-1.7.3+0.x64\share\julia\stdlib\v1.7\Sockets\src\Sockets.jl:532    
 [2] connect
   @ C:\Users\Braam\.julia\juliaup\julia-1.7.3+0.x64\share\julia\stdlib\v1.7\Sockets\src\Sockets.jl:567 [inlined]  
 [3] connect
   @ C:\Users\Braam\.julia\juliaup\julia-1.7.3+0.x64\share\julia\stdlib\v1.7\Sockets\src\PipeServer.jl:97 [inlined]
 [4] serve(args::String; is_dev::Bool, crashreporting_pipename::String)
   @ VSCodeServer c:\Users\Braam\.vscode\extensions\julialang.language-julia-1.6.30\scripts\packages\VSCodeServer\src\VSCodeServer.jl:110
 [5] top-level scope
   @ c:\Users\Braam\.vscode\extensions\julialang.language-julia-1.6.30\scripts\terminalserver\terminalserver.jl:39
in expression starting at c:\Users\Braam\.vscode\extensions\julialang.language-julia-1.6.30\scripts\terminalserver\terminalserver.jl:14
```

**Comment by pfitzseb** [comment]
Fixed in 1.6.31.
